### ISSUE TITLE
[Bug Report] Vue Warning when install Vuetify

### ISSUE THREAD

**Issue by Ku4nCheang** [vuetifyjs/vuetify#2803]
### Versions and Environment
**Vuetify:** 0.17.4
**Vue:** 2.5.11
**Browsers:** Chrome 63.0.3239.84
**OS:** Mac OS 10.13.2

### Steps to reproduce
1. Run the code
2. Open developer console.
3. You can see a Vue warning about "Invalid key "validate" in validation rules object for prop "activator"."

### Expected Behavior
No warning should occur

### Actual Behavior
warning occur 

### Reproduction Link
[http://jsfiddle.net/h4kpvk4v/](http://jsfiddle.net/h4kpvk4v/)

**Comment by jacekkarczmarczyk** [comment]
It's fixed in 1.0.0-alpha.2, if you don't want to upgrade vuetify you can downgrade vue to 2.5.9

**Comment by motchju** [comment]
Thanks !

**Comment by phil1996s** [comment]
Hey there,
I've already installed 1.0.0-alpha.3 and the error accures nevertheless.

**Comment by johnleider** [comment]
Do you have a codepen producing the issue in alpha3?

**Comment by phil1996s** [comment]
Because the project is just minutes old, I've had only this to things edited

main.js:
```import Vue from 'vue'
import App from './App'
import router from './router'
import Vuetify from 'vuetify'
import colors from 'vuetify/es5/util/colors'

import('../node_modules/vuetify/dist/vuetify.min.css')

Vue.use(Vuetify)

Vue.config.productionTip = false

/* eslint-disable no-new */
new Vue({
  el: '#app',
  router,
  template: '<App/>',
  components: { App }
})´´´

App.vue:
´´´
<template>
  <v-app>
    <v-toolbar class="blue darken-2" dark>
      <v-toolbar-side-icon class="white--text"></v-toolbar-side-icon>

      <v-toolbar-title>{{ title }}</v-toolbar-title>

      <v-spacer></v-spacer>

      <v-btn icon>
        <v-icon>more_vert</v-icon>
      </v-btn>
    </v-toolbar>
    <router-view/>
  </v-app>
</template>

<script>
//import languages
import languages from '@/env/languages.js'

export default {
  name: 'app',
  data() {
    return {
      language: '',
      title: '',
    }
  },
  created() {
    this.language = languages[localStorage.getItem('userLanguage')],
    this.title = this.$route.name
  }
}
</script>
´´´

**Comment by Laurence88** [comment]
Hi,

Hi also have the same issue.
I am using "vue": "^2.5.11",  "vuetify": "^1.0.0-Alpha.3".x
![x](https://user-images.githubusercontent.com/34590786/34066958-ca7ace84-e264-11e7-96c8-a99bc2c794ab.png)

**Comment by KaelWD** [comment]
Can you guys both `console.log(Vuetify.version)`?

**Comment by Laurence88** [comment]
yes, done and it says vuetify@1.0.0-alpha.1
Yet it is 1.0.0-alpha.3 in package.json so obviously I did not have the version I though I had installed :)

![screen shot 2017-12-16 at 2 10 41 pm](https://user-images.githubusercontent.com/34590786/34067229-fe8536b4-e26a-11e7-9f93-773939176ceb.png)

**Comment by Laurence88** [comment]
I have updated to version alpha-3 properly this time and it is working now! 
No more error message.
Sorry for the inconvenience. 
Thanks for your great work!!! Have a good week-end!

**Comment by mizzao** [comment]
Seems to be fixed on 0.17.5 as well.
